cfme/networks/network_router.py

```diff
@@ -55,6 +56,15 @@ def add_interface(self, subnet_name):
                                                                             router=self.name)
         view.flash.assert_success_message(success_msg)
 
+    def add_subnet(self, subnet_name, address, netmask, gateway):
+        view = navigate_to(self, 'AddSubnet')
+        view.fill({'subnet_name': subnet_name,
+                   'address': address,
+                   'netmask': netmask,
+                   'gateway': gateway})
+        view.add.click()
+        view.flash.assert_no_error()
```

Comment:
For the record this would be a good pattern to push to the method above, `add_interface`, and the CRUD methods for this class.   Make specific flash message assertions in the test using this method, just make `assert_no_error()` calls in the method itself.

Reply:
To be clear - does not need to be changed in this PR, can be a separate enhancement PR.

Reply:
I think I would rather create a new PR for this, since it's an enhancement related to other entities. :)